modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -605,8 +273,9 @@ class Instrument_Manager extends \NDB_Menu_Filter
      */
     protected function isAdminUserConfigured() : bool
     {
-        $db        = $this->_factory->database();
-        $db_config = $this->_factory->config()->getSetting('database');
+        $factory   = \NDB_Factory::singleton();
+        $db        = $factory->database();
```

Comment:
The database should come from $this->loris->getDatabaseConnection(), there's no reason to use the factory singleton.